src/dipro.f90

```diff
@@ -0,0 +1,483 @@
+! This file is part of dipro.
+! SPDX-Identifier: Apache-2.0
```

Comment:
Shall we keep the routines for dipro under Apache-2.0 license or shall we adopt the same license as for xtb.

Reply:
Do as you wish. I don't know anything about licenses.